packages/notte-agent/src/notte_agent/gufo/agent.py

```
@@ -1,198 +1,45 @@
+from notte_agent.agent import NotteAgent
+from notte_agent.common.types import AgentStepResponse
 from notte_agent.gufo.perception import GufoPerception
 from notte_agent.gufo.prompt import GufoPrompt
 
 
 class GufoConfig(NotteConfig):
     enable_perception: bool = True
-    auto_scrape: bool = True
 
     @field_validator("enable_perception")
     def check_perception(cls, value: bool) -> bool:
```

Pylint (3.3.7)

[error] Method 'check_perception' should have "self" as first argument

(E0213)